doc-content/jbpm-docs/src/main/asciidoc/BPMN2/bpmn-subprocesses-ref.adoc

```diff
@@ -0,0 +1,74 @@
+
+* *Called Element*: The ID of the subprocess that the activity calls and instantiates.
+* *Independent*: If selected, the subprocess is started as an independent process. If not selected, the active subprocess is canceled when the parent process is terminated.
+* *Abort Parent*: If selected, non-independent reusable subprocesses can abort the parent process when there is an error during the execution of the called process instance. For example, when there's an error when trying to invoke the subprocess or when the subprocess instance is aborted. This property is visible only when the *Independent* property is not selected. The following rules apply:
+** If the reusable subprocess is independent, *Abort parent* is not available.
+** If the reusable subprocess is not independent, *Abort parent* is available.
+* *Wait for completion*: If selected, the specified *On Exit Action* is not performed until the called subprocess instance is terminated. The parent process execution continues when the *On Exit Action* completes. This property is selected (set to `true`) by default.
+* *Is Async*: Select if the task should be invoked asynchronously and cannot be executed instantly.
+* *Multiple Instance*: Select to execute the subprocess elements a specified number of times. If selected, the following options are available:
+** *MI Execution mode*: Indicates if the multiple instances execute in parallel or sequentially. If set to *Sequential*, new instances are not created until the previous instance completes.
+** *MI Collection input*: Select a variable that represents a collection of elements for which new instances are created. The subprocess is instantiated as many times as the size of the collection.
+** *MI Data Input*: Specifies the name of the variable containing the selected element in the collection. The variable is used to access elements in the collection.
+** *MI Collection output*: Optional variable that represents the collection of elements that will gather the output of the multi-instance node.
+** *MI Data Output*: Specifies the name of the variable that is added to the output collection that you selected in the *MI Collection output* property.
+** *MI Completion Condition (mvel)*: MVEL expression that is evaluated on each completed instance to check if the specified multiple instance node can complete. If it evaluates to `true`, all remaining instances are canceled.
+* *On Entry Action*: A Java or MVEL script that specifies an action at the start of the task.
+* *On Exit Action*: A Java or MVEL script that specifies an action at the end of the task.
+* *SLA Due Date*: The date that the service level agreement (SLA) expires.
+
+.Reusable subprocess properties
+image::BPMN2/subproc-props.png[A screenshot of Subprocess properties]
+
+[[_event_sub_process]]
+.Event subprocess
+An event subprocess becomes active when its start event is triggered. It can interrupt the parent process context or run in parallel with it.
```

Comment:
Suggestion - can the translate-ability be improved? also, passive voice?

`Active` is referred to in IBMSG 301p but we use this throughout our docs. Maybe use "activates" and "triggers"?

s/An event subprocess becomes active when its start event is triggered./An event subprocess activates when its start event triggers.

Reply:
@csherrar, here the word `active` indicates the state of the subprocess. The other states are pending, aborted, and so on. So I believe we can keep it as is. 
wdyt?